client/python/cryoet_data_portal/src/cryoet_data_portal/_model_stubs.py

```diff
@@ -0,0 +1,191 @@
+"""Contains model class stubs to define manually maintained methods."""
+from __future__ import annotations
+
+import os
+from typing import Iterable, Optional
+
+from cryoet_data_portal._file_tools import download_directory, download_https
+
+
+class Annotation:
+    files: Iterable[AnnotationFile]
```

Comment:
These are only needed for type checking the method bodies. We should probably compare the parsed fields to verify the typing of these is correct.